### PR TITLE
remote wrappers

### PR DESCRIPTION
**Pull request by Pegmode**
Wrappers for connecting our api to the remote nodes

### CODE REVIEW

**Comment by Pegmode** [comment]
Tons of intermittent outages in the remote node's API's is causing quite a bit of slowdown. I've gotten most of the `GET`s for post but some of the endpoints seem broken. Hard to tell with the outages and lack of documentation for one group.

File: server/utils/remote_constants.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REMOTE_NODE1_HOST = "https://deployservertest.herokuapp.com/"
+REMOTE_NODE2_HOST = "https://website404.herokuapp.com/"
```
Comment (giancarlopernudisegura) [inline server/utils/remote_constants.py]: We have the remote node DB table for this

Comment (Pegmode) [reply server/utils/remote_constants.py]: I did this because there was a minor difference between the 2, I can move it back to pulling it from the db.

Comment (Pegmode) [reply server/utils/remote_constants.py]: addressed in https://github.com/giancarlopernudisegura/cmput404/pull/128/commits/2b0e20bd4f0a1334a8ec368befac9e3bd80a76ee

File: server/exts.py [hunk 2]
```diff
@@ -10,6 +10,8 @@
 
 LOCAL_AUTH_USER = os.getenv("LOCAL_AUTH_USER")
 LOCAL_AUTH_PASSWORD = os.getenv("LOCAL_AUTH_PASSWORD")
+<<<<<<< HEAD
```
Comment (dalidia) [inline server/exts.py]: Can you fix this? this is coming from merge conflicts

File: server/exts.py [hunk 3]
```diff
@@ -10,6 +10,8 @@
...
+=======
```
Comment (dalidia) [inline server/exts.py]: merge conflicts again

File: server/routes/api.py [hunk 4]
```diff
@@ -276,9 +326,25 @@ def get_comments(author_id: str, post_id: str) -> Response:
     methods=["GET"],
 )
 def get_comment(author_id: str, post_id: str, comment_id: str) -> Response:
+<<<<<<< HEAD
```
Comment (dalidia) [inline server/routes/api.py]: merge conflicts as well here

**Review by dalidia — CHANGES_REQUESTED** [review]
Can you fix the merge conflicts?

**Review by dalidia — APPROVED** [review]
Looks good!

**Pull request merged**